extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -288,4 +288,14 @@
         </testResources>
     </build>
 
+    <profiles>
+        <profile>
+            <id>spark-3.0</id>
```

Comment:
spark 3.0 is EOL after #2974 . Maybe we can use a command line argument to specify the Spark version. WDYT? cc @pan3793

Reply:
sgtm